.github/workflows/release.yml

```diff
@@ -6,17 +6,23 @@ on:
       - 'v*.*.*'
     branches:
       - main
+    paths-ignore:
+      - '**/*.md'
+      - '.gitignore'
+      - '.github/**'
+      - '!.github/workflows/release.yml'
+      - 'docker/**'
   workflow_dispatch:
 
 env:
-  cache_nonce: 0 # Allows for easily busting actions/cache caches
-  nim_version: pinned
-  rust_version: 1.79.0
+  nim_version:   2.2.4
+  rust_version:  1.79.0
+  cmake_version: 3.x
   archivist_binary_base: archivist
   cirdl_binary_base: cirdl
   build_dir: build
-  nim_flags: ''
-  windows_libs: 'libstdc++-6.dll libgomp-1.dll libgcc_s_seh-1.dll libwinpthread-1.dll'
+  nim_flags: '-d:release -d:danger'
```

Comment:
I would not add `-d:danger`, it disables a lot of runtime checks, such as overflows and boundary checking of arrays and seqs. It was also not present in the nimbus build that we had before.